### PR TITLE
Remove flush($argument) from documentation as it's deprecated.

### PR DESCRIPTION
**Pull request by Warxcell**

### CODE REVIEW

**Review by SenseException — CHANGES_REQUESTED** [review]
Thanks for pointing this out. I'd say that the 2.9,x branch should mention its deprecation while it should be removed from the docs in 3.0.x

File: docs/en/reference/unitofwork.rst [hunk 1]
```diff
@@ -134,6 +134,10 @@ optimize the performance of the Flush Operation:
   explicit strategies of notifying the UnitOfWork what objects/properties
   changed.
 
+.. warning::
```
Comment (SenseException) [inline docs/en/reference/unitofwork.rst]:
```suggestion
.. note::
```

Because `note` was used in other deprecation boxes.

**Review by SenseException — CHANGES_REQUESTED** [review]

File: docs/en/reference/unitofwork.rst (hunk 1 above)
Comment (Warxcell) [reply docs/en/reference/unitofwork.rst]: Not quite I would say. 3 of 10 total deprecations are with warning.

image
image
image

Comment (Warxcell) [reply docs/en/reference/unitofwork.rst]: I refactored all other deprecations to note, for sake of consistency.

**Review by SenseException — APPROVED** [review]
Thank you for fixing the other inconsistencies.

**Pull request merged**